plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -439,10 +442,14 @@ export class UserInteractionInstrumentation extends InstrumentationBase<unknown>
         applyThis?: any,
         applyArgs?: any
       ): Zone {
-        const target: HTMLElement | undefined = task.target;
+        const event =
+          Array.isArray(applyArgs) && applyArgs[0] instanceof Event
+            ? applyArgs[0]
+            : undefined;
+        const target = event?.target;
         let span: api.Span | undefined;
         const activeZone = this;
-        if (target) {
+        if (target instanceof HTMLElement) {
```

Comment:
See previous comment - if `_createSpan` does this check, it doesn't need to be repeated elsewhere.